dbms/src/Flash/executeQuery.cpp

```diff
@@ -199,6 +199,7 @@ std::optional<QueryExecutorPtr> executeAsPipeline(Context & context, bool intern
     if (likely(!internal))
         LOG_INFO(logger, fmt::format("Query pipeline:\n{}", executor->toString()));
     dag_context.switchToPipelineMode();
+    dag_context.setEnableResourceControl(context.getSettingsRef().enable_resource_control);
```

Comment:
```suggestion
    dag_context.setEnableResourceControl(true); or dag_context.enableResourceControl();
```
because `enable_resource_control` must be true here.

Reply:
done